logrotate/osmap.yaml

```
@@ -4,3 +4,4 @@
 Ubuntu:
   default_config:
     su: root syslog
+    login_records_jobs: false
```

`login_records_jobs` needs to be defined at the same level as `default_config`:

```suggestion
  login_records_jobs: false
  default_config:
    su: root syslog
```

This can be seen by the other examples in the formula:

* https://github.com/saltstack-formulas/logrotate-formula/search?q=login_records_jobs

Also shows up in the CI debug output (https://gitlab.com/saltstack-formulas/logrotate-formula/-/jobs/2119150344#L1260):

```sls
[DEBUG   ] ### MAP.JINJA DUMP ###
values:
  ...
  default_config:
    ...
    login_records_jobs: false
  ...
  login_records_jobs: true
  ...
```